monkestation/code/modules/bloodsuckers/bloodsucker/bloodsucker_sol.dm

```diff
@@ -86,15 +62,15 @@
 
 	switch(danger_level)
 		if(DANGER_LEVEL_FIRST_WARNING)
```

Comment:
do you know how hard it is to avoid bringing up project moon brainrot with this- wait i just did it